bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/Encrypt.cs

```diff
@@ -31,10 +31,7 @@ public class EncryptionDevice : DisposableBase
         {
             var status = NativeInterface.EncryptionDevice.New(
                 deviceUuid, sessionUuid, launchCode, location, out Handle);
-            if (status != Status.ELECTIONGUARD_STATUS_SUCCESS)
-            {
-                Console.WriteLine($"EncryptionDevice New Error Status: {status}");
-            }
+            status.ThrowIfError();
```

Comment:
this should be applied anywhere the status is checked, but only did it in files i was already touching for now